sourcegraph/sourcegraph
# (feat) add autocompletion for repo meta creation page

Part of https://github.com/sourcegraph/pr-faqs/issues/96.

## Test plan
- `sg start`
- Enable `repository-metadata` feature flag
- Open any repo page repo meta setting page, example: https://sourcegraph.test:3443/github.com/sourcegraph/sourcegraph/-/settings/metadata
- Check that key input suggests similar existing repo metadata when typing

## Screenshot

https://github.com/sourcegraph/sourcegraph/assets/6717049/b8e0c1f7-7626-4e71-ac2e-e181b3028536
